cli/i18n/I18nFile.php

```diff
@@ -80,7 +79,7 @@ private function flatten($translation, $prefix = '') {
 	private function unflatten($translation) {
 		$a = array();
 
-		ksort($translation);
+		ksort($translation, SORT_NATURAL | SORT_FLAG_CASE);
```

Comment:
@Frenzie I've sorted using natural order.

Reply:
Requires PHP 5.4 (which is all perfectly fine now). :-)